en/docs/secure-health-apis/guides/getting-patientid.md

```diff
@@ -0,0 +1,96 @@
+# Retrieving the Patient ID
```

Comment:
Should be something like "Configuring SMART App Launch for a patient user"

Reply:
Fixed in https://github.com/wso2/open-healthcare-docs/pull/44/commits/265eb2611a3f4cda6c4e230c05fb3f442d5afcee